requirements.in

```diff
@@ -1,13 +1 @@
-aiokatcp>=1.5.0
-dask
-katsdpservices[aiomonitor]>=1.2
-katsdpsigproc[CUDA]>=1.6.1
-katsdptelstate
-numpy
-prometheus-async[aiohttp]
 psutil  # Gives more accurate dask.system.CPU_COUNT
```

Comment:
It's not needed to run the package in most cases. We just want it in the deployed image because it makes dsim behave better when it's pinned to a limited set of cores.

Reply:
Right, that makes sense.